integration-tests-support/test-support/src/main/java/org/apache/camel/quarkus/test/AvailablePortFinder.java

```diff
@@ -103,4 +124,19 @@ private static boolean isQuarkusReservedPort(int port) {
         }
         return false;
     }
+
+    private static String getCallerClassName() {
+        return StackWalker.getInstance(StackWalker.Option.RETAIN_CLASS_REFERENCE)
+                .walk(s -> s.map(StackWalker.StackFrame::getClassName)
+                        .filter(className -> !className.equals(AvailablePortFinder.class.getName()))
```

Comment:
Shouldn't we make AvailablePortFinder class final, to better match the assumption made on the above line?

Reply:
> Shouldn't we make AvailablePortFinder class final

The class declaration is:

```java
public final class AvailablePortFinder {
}
```

Reply:
Filtering on className should be enough then.

Reply:
> The class declaration is:
> 
> ```java
> public final class AvailablePortFinder {
> }
> ```

I am blind. Sorry for the noise.